.eslintrc

```diff
@@ -0,0 +1,11 @@
+env:
+  node: true
+
+rules:
+  comma-dangle: 0
+  curly: 0  
+  key-spacing: 0
+  no-use-before-define: [1, nofunc]
+  quotes: [1, single]
+  semi: 0
+  strict: 0
```

Comment:
I didn't install ESLint as a devDep or anything, but if you have it installed globally, it currently spits out two errors:

```
$ eslint .

bin/seagull.js
   76:4   error  args is defined but never used                  no-unused-vars
  126:10  error  content is already declared in the upper scope  no-shadow

✖ 2 problems (2 errors, 0 warnings)
```